app/services/description_import.rb

```diff
@@ -68,4 +68,10 @@ def key?(what, key)
       what.key?(key)
     end
   end
+
+  def compact_params(params)
+    params.each do |key, value|
```

Comment:
Does this only work at the top level?  Is that good enough or do we need to ensure this happens at any level of the tree?

Reply:
Good catch. Not sure how likely that is, but this small change makes it recursive and I've updated the description above.